runtime/doc/usr_05.txt

```diff
@@ -245,7 +246,46 @@ The ":map" command (with no arguments) lists your current mappings.  At
 least the ones for Normal mode.  More about mappings in section |40.1|.
 
 ==============================================================================
-*05.4*	Adding a plugin					*add-plugin* *plugin*
+*05.4*	Adding a package			*add-package* *matchit-install*
+
+A package is a set of files that you can add to Vim.  There are two kinds of
+packages: optional and automatically loaded on startup.
+
+The Vim distribution comes with a few packages that you can optionally use.
+For example, the matchit plugin.  This plugin makes the "%" command jump to
+matching HTML tags, if/else/endif in Vim scripts, etc.  Very useful, although
+it's not backwards compatible (that's why it is not enabled by default).
+
+To start using the matchit plugin, add one line to your vimrc file: >
+	packadd matchit
```

Comment:
Perhaps we should disable vimball by default, and use that as the example instead.